### DISCUSSION TITLE
Card for GitHub Gist

### DISCUSSION THREAD

**Discussion by VydrOz** [anuraghazra/github-readme-stats#3044] [Ideas]
For the moment, the cards only support repos and not GitHub Gist files.

To be honest, I found someone who did, but with only 2 themes. (https://github.com/Yizack/gists-readme)

**Comment by qwerty541** [comment]
I'm not sure if card with list of gists names make any sense

![image](https://github.com/anuraghazra/github-readme-stats/assets/53787217/c911cb5f-20de-496b-ac1b-a18735433929)

But it think that card with information about one gist can be a good addition to github-readme-stats. Also we can add optional ability to show count of gists published by user on our stats card.

![image](https://github.com/anuraghazra/github-readme-stats/assets/53787217/67e6f8d3-133a-4120-b42c-8fb830a51b98)

@rickstaa What do you think about that?

**Reply by rickstaa** [reply]
@qwerty541 I agree with your conclusion. I think a list of gists doesn't make much sense since people can see a user's gists on their profile when they want that information (see https://gist.github.com/rickstaa). Adding the ability to also list gists for the pinned card would make sense and would be something I would approve a PR for 👍🏻.

**Reply by qwerty541** [reply]
@rickstaa @VydrOz I have opened an issue https://github.com/anuraghazra/github-readme-stats/issues/3047  with feature request from this discussion, so it can be closed now.

**Reply by VydrOz** [reply]
Thank you for your answers and your responsiveness! I also agree with this conclusion 👍

**Reply by qwerty541** [reply]
@rickstaa @VydrOz I have opened draft pull request https://github.com/anuraghazra/github-readme-stats/pull/3064, please tell me what do you think.

**Reply by rickstaa** [reply]
Merged into master 🚀!

```md
[![Gist Card](https://github-readme-stats.vercel.app/api/gist?id=bbfce31e0217a3689c8d961a356cb10d)](https://gist.github.com/Yizack/bbfce31e0217a3689c8d961a356cb10d/)
```

[![Gist Card](https://github-readme-stats.vercel.app/api/gist?id=bbfce31e0217a3689c8d961a356cb10d)](https://gist.github.com/Yizack/bbfce31e0217a3689c8d961a356cb10d/)
